doc/manual/config-advanced.rst

```diff
@@ -23,12 +23,13 @@ path of your installation) as follows:
 - *Team pictures*: a photo of the team will be shown in the team details
   page if `public/images/teams/456.jpg` exists, where *456* is the
   team's numeric ID as shown in the DOMjudge interface.
-- *Banner*: a page-wide banner can be shown on the public scoreboard
-  if that image is placed in `public/images/banner.png`.
+- *Contest Banners*: a page-wide banner can be shown on the public scoreboard
+  if that image is placed in `public/images/banners/1.png` where *1* is the
+  contest's numeric ID as shown in the DOMjudge interface.
```

Comment:
There's no file that applies to all?

Reply:
Nope, I'd say if we improve our import-contest.sh to also read the banner while importing the contest that is not a big problem. Or do you disagree? We could add a fallback to `images/banners/global.png` or `images/banner.png`.

Reply:
I actually do disagree. Using `import-contest` is just one of way setting up a contest and the banner will be the same in most situations across contests. So basically having a fallback banner is less work and trivial to implement.

Reply:
+1. I've used this at our company prelim round, and I find it very convenient that I can just upload one (default) banner.

Reply:
It was not 100% trivial (since I think we don't want to be able to delete the global banner using the contest edit form) but I added it back.